pkg/helpers/authchallengers/negotiator_sspi.go

```diff
@@ -246,19 +246,15 @@ func (s *sspiNegotiator) getPassword(domain, username string) (string, error) {
 	password := s.password
 
 	if missingPassword := len(password) == 0; missingPassword {
+		if len(s.webConsoleURL) > 0 {
```

Comment:
nit: we are changing the logging order from;
```go
fmt.Fprintf(s.writer, "Authentication required for %s (%s)\n", s.host, domain)
fmt.Fprintf(s.writer, "Console URL: %s/console\n", s.host)
```
to
```go
fmt.Fprintf(s.writer, "Console URL: %s/console\n", s.host)
fmt.Fprintf(s.writer, "Authentication required for %s (%s)\n", s.host, domain)
```

Reply:
yes, I believe that this is the proper order. I'd rather not have text in between "Authentication required" and the password prompt.